Makefile

```diff
@@ -221,7 +221,7 @@ endif
 
 deploy: IMAGE_TAG?=$(shell if [ "${RELEASE_TAG}" == "" ]; then echo "${IMAGE_TAG}"; else echo "${RELEASE_TAG}"; fi;)
 deploy:
-	@ecs deploy $(ENVIRONMENT) impact --image web $(DOCKER_REGISTRY)/impact-api:$(IMAGE_TAG) --image redis $(DOCKER_REGISTRY)/redis:$(IMAGE_TAG)
+	@ecs deploy --ignore-warnings $(ENVIRONMENT) impact --image web $(DOCKER_REGISTRY)/impact-api:$(IMAGE_TAG) --image redis $(DOCKER_REGISTRY)/redis:$(IMAGE_TAG)
```

Comment:
Which warnings are we ignoring? Is this something to be worried about re security and/or future deployment safety?

Reply:
I'm curious about what `--ignore-warnings` is ignoring as well. I presume that since they're warnings (not errors) it's not terribly risky, but if it's simple and concrete it might be worth a comment in the Makefile.

Reply:
currently this ignores a timeout issue which is a known issue triggered by our current setup (single instance cluster) https://github.com/fabfuel/ecs-deploy/issues/11

unfortunately for the time being i dont think there is a reasonable workaround for this - however, i do not see any security issues being swallowed up by this unless you know of any specific cases.